### PR TITLE
Manuscript

### PR DESCRIPTION
**Pull request by wincowgerDEV**
Hoping all coauthors can review the manuscript by September 19th. Let me know if you need more time.

### CODE REVIEW

**Comment by wincowgerDEV** [comment]
I would like to add two other coauthors to the manuscript from Let's Do It World (@KristiinaKerge and @krishaamer). They were a huge part of the initial push for [WADE AI](https://github.com/letsdoitworld/wade-ai) which inspired this project and they are interested in helping us review the manuscript and help us make the tool have a global presence.

File: paper.md
Comment (feydan) [inline paper.md]: > average data scientist

should this be `littler researcher`?  I feel like data scientists would be the ones who know how to run models.

> and serverless architecture (e.g. AWS Lambda)

Currently lambda isn't really doing anything special or enabling any workflows that wouldn't be possible with other app hosting methods.  We are primarily using it right now because it is currently the cheapest way to host our simple back-end application (most of the logic is on the front-end).

File: paper.md
Comment (feydan) [inline paper.md]: > Any data uploaded to the platform is automatically saved to an

may be automatically saved   -- I think this is currently disabled, but we may re-enable it in the future.

File: paper.md
Comment (feydan) [inline paper.md]: Code for Sacramento and Open Fresno

Comment (feydan) [inline paper.md]: Might also be worth mentioning that there are some instructions (though not exhaustive) for [setting it up and hosting it via your own AWS account](https://github.com/code4sac/trash-ai/blob/manuscript/docs/git-aws-account-setup.md) if anyone was interested in forking, modifying, and/or white labeling the application.  Our deployment model also [uses github actions](https://github.com/code4sac/trash-ai/blob/manuscript/docs/github-actions.md), so the CI/CD setup is built into the code.  These concerns are non-trivial to solve for any team that is looking maintain a project like this with multiple contributors.

File: paper.md
Comment (feydan) [inline paper.md]: Code for Sacramento and Open Fresno

Comment (feydan) [inline paper.md]: We acknowledge the work of the Code for Sacramento and Open Fresno teams, brigades of Code for America

^ Code for Sacramento and Open Fresno are not actually legally a 'part' of Code for America.  We are separate legal entities that have a partnership with Code for America via a Memorandum of Understanding.

File: paper.md
Comment (feydan) [inline paper.md]: Code for Sacramento and Open Fresno

**Review by feydan — CHANGES_REQUESTED** [review]

File: paper.md
Comment (feydan) [inline paper.md]: Gina Durante

File: paper.md
Comment (feydan) [inline paper.md]: Brianda Hernandez

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: Updated to litter researcher, good idea. 

Added the word affordable for the second part, so that it is clear that is why we are going serverless.

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: updated with your recs https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d

**Review by feydan — APPROVED** [review]

File: paper.md
Comment (shollingsworth) [inline paper.md:11]: Since we're discussing specific tech I'd suggest adding
* vue
* vuetify
* vite
* pinia

**Review by shollingsworth — APPROVED** [review]
looks good 👍🏻

**Review by walteryu — APPROVED** [review]
Manuscript looks great - my only comment is for the limitations section: One limitation based on user feedback is the distance of trash from the camera. Since the AI model was trained using the TACO dataset which consists of images of trash within a reasonable distance, it performs less accurately on images with trash which is farther away such as when taken from a vehicle.

**Comment by maryfnorris** [comment]
I did notice that Democracy Lab is listed twice. Otherwise,  this all looks good. Thank you for acknowledging Open Fresno.

**Review by hernandezforges — COMMENTED** [review]
No comments.

**Comment by wincowgerDEV** [comment]
> I did notice that Democracy Lab is listed twice. Otherwise, this all looks good. Thank you for acknowledging Open Fresno.

Thanks @maryfnorris for the review and for finding that. Correction made.

**Comment by wincowgerDEV** [comment]
> No comments.

Thanks @createHernandez for the review and for the positive response.

File: paper.md
Comment (wincowgerDEV) [reply paper.md:11]: Thanks @shollingsworth got these incorporated. https://github.com/code4sac/trash-ai/commit/3652a152ea3d3afb7e6df823f0791b1bb0e081a4

**Comment by wincowgerDEV** [comment]
> Manuscript looks great - my only comment is for the limitations section: One limitation based on user feedback is the distance of trash from the camera. Since the AI model was trained using the TACO dataset which consists of images of trash within a reasonable distance, it performs less accurately on images with trash which is farther away such as when taken from a vehicle.

Thanks Walter, I have added this context into https://github.com/code4sac/trash-ai/commit/39726ed2705bfc3580c8e8dc7289484a85c21e59

**Review by maryfnorris — APPROVED** [review]

**Comment by maryfnorris** [comment]
We have the green light to add Brad Andersen as a contributor.

**Comment by wincowgerDEV** [comment]
> We have the green light to add Brad Andersen as a contributor.

Great to have Brad on board, can we add him as a contributor so that he can do his review of the manuscript?

File: README.md
Comment (gdurante2019) [inline README.md]: First sentence--suggest placing comma after "litter" and before "which".

Comment (wincowgerDEV) [reply README.md]: updated! Thanks for the recommendation.

**Comment by maryfnorris** [comment]
Absolutely!

Mary C Norris

**Review by gdurante2019 — APPROVED** [review]

File: paper.md
Comment (gdurante2019) [inline paper.md]: Second sentence--suggest changing to read as follows:  "...classification of **individual items of trash within images**, hosted at www.trashai.org." (Bolded for emphasis.)  Since the next sentence begins by stating that images can be uploaded as a batch, it's not necessary to state that in the previous sentence, and the original wording was a bit awkward.

File: paper.md
Comment (gdurante2019) [inline paper.md]: First sentence:  Delete "The" and begin the sentence with "Trash is..."

Sentence 10--put a comma after each 'e.g.', as follows:  "New developments in browser-side AI (e.g., tensorflow.js) and serverless architecture (e.g., AWS Lambda)"

Sentence 10--Put a comma after 'GUI', as follows:  "...affordable browser-side artificial intelligence in a web GUI, alleviating both obstacles."

Section is excellent otherwise--very clear!

File: paper.md
Comment (gdurante2019) [inline paper.md]: Last sentence--suggest adding comma after link to S3 and before the word 'which', as follows:  "...saved to an [S3 bucket](https://aws.amazon.com/s3/), which we can use to improve the model over time."

File: paper.md
Comment (gdurante2019) [inline paper.md]: Add a comma before the word 'which' in the first and second sentences, as follows:  

First sentence:  "...starting with the TACO dataset, which was available with a complimentary Jupyter Notebook..."

Second sentence:  "...An example notebook was referenced, which used the default `YOLO v5 model`..."

File: paper.md
Comment (gdurante2019) [inline paper.md]: Sentence 3--suggest comma after 'e.g.':  "...not be considered trash (e.g., a drink bottle on someone's desk vs in the forest..."

Sentence 6--suggest hyphen for 'long term':  "...make a general trash AI a challenging (yet worthwhile) long-term endeavor."

Sentence 7--suggest rewriting as follows:  "The algorithm is primarily trained on single pieces of trash in the image, with the trash laying on the ground; thus, model class prediction of trash in these kinds of images will generally be better than, say, trash appearing in aerial images."  (Note--I wouldn't say that the current model 'excels' at correctly classifying trash in this or any other use case, but rather, the accuracy of its classifications will be better for these kinds of images than other types of images containing trash.)

File: paper.md
Comment (gdurante2019) [inline paper.md]: Sentence 4--capitalize 'code' in 'brigades of code for America'; put a comma after the word 'possible' and before the word 'and', as follows:  "We acknowledge the work of the Code for Sacramento and Open Fresno team, brigades of **C**ode for America, without whom this project would not have been possible, and acknowledge the input of the California Water Monitoring Council Trash Monitoring Workgroup."

Sentence 6--add comma after "In particular" and add the word "and" after "...Joseph Falkner, Democracy Lab,..." to make it clear that the names that follow are all associated with the University of Houston, as follows:  "In particular, we would like to acknowledge Joseph Falkner, Democracy Lab, **and** Brad Anderson, Jim Ewald, **and** Don Brower **of the** University of Houston. We acknowledge financial support from McPike Zima Charitable Foundation."

**Review by gdurante2019 — CHANGES_REQUESTED** [review]
Please review and incorporate suggested changes, as outlined in the comments.

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: https://github.com/code4sac/trash-ai/commit/3a33857c8ff03c2b308c58b3b5a7bda3a3a79ba9

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: I want to leave in the "In the environment part because there needs to be a distinction between what we are doing and waste management AI which is a whole different problem that I am not sure this addresses.

Comment (wincowgerDEV) [reply paper.md]: Made other recommended changes in https://github.com/code4sac/trash-ai/commit/01e6bad6f97ffc431a635a7883476e12524ac3f0

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: Made change https://github.com/code4sac/trash-ai/commit/14eb61cb7ef35ff83a27bfc045718aef1ecbb60e

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: Change made. https://github.com/code4sac/trash-ai/commit/1654df8a4e2224fa744d99d8fcafd30b2587e773

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: Edits made in https://github.com/code4sac/trash-ai/commit/0b4743ed252ad7f7351950aecef4c1c77e8e7aa0. Split up your recommendation for sentence 7 a little so that it wouldn't be as long.

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: Sentence 4 recommendation made. https://github.com/code4sac/trash-ai/commit/21ad88f5d61a83cf08310bd2b93712986c5b13a9

Sentence 6, I might be a little confused are you saying Brad Anderson and Jim Ewald are with Democracy lab?
I am not sure if Don Brower is of the University of Houston, @walteryu can you confirm?

Comment (gdurante2019) [reply paper.md]: Correction--after talking with Walter during the Code4Sac meetup this evening, please replace the original sentence with the following:

"In particular, we would like to acknowledge Joseph Falkner, Democracy Lab, Brad Anderson, Jim Ewald, Don Brower, and University of Houston.”

The only changes from the original sentence are the comma after "particular" and the word "and" before "University of Houston".

**Comment by wincowgerDEV** [comment]
> Please review and incorporate suggested changes, as outlined in the comments.

All of your comments have been incorporated Gina :) Thanks for the great feedback.

**Comment by wincowgerDEV** [comment]
@feydan I have gone through and made all requested changes to the manuscript. Could you merge the PR to main? Once it is merged to the main, I can go ahead and submit the manuscript :)

**Comment by wincowgerDEV** [comment]
@gdurante2019 just making sure before we merge, did I address all your comments?

**Comment by wincowgerDEV** [comment]
@shollingsworth not sure who has permission to merge this pull request but I believe everything has been addressed and it is ready to be merged.

**Review by shollingsworth — APPROVED** [review]

**Comment by shollingsworth** [comment]
> @shollingsworth not sure who has permission to merge this pull request but I believe everything has been addressed and it is ready to be merged.

@gdurante2019 needs to re-review, once she hits approve we should be able to merge.

File: paper.md
Comment (gdurante2019) [inline paper.md]: Delete "The" at the beginning of first sentence, and start the sentence with "Trash"

File: paper.md
Comment (gdurante2019) [inline paper.md]: Second sentence, first word:  Put quotation marks around Trash, e.g., 'Trash' is a word people use...

**Review by gdurante2019 — APPROVED** [review]
I've suggested a couple of minor edits as comments.  Please check those out and decide whether or not you'd like to incorporate.  Thanks!

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: https://github.com/code4sac/trash-ai/commit/640e465f6b904b30308889c2a8fa56228e9c5fd0

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: https://github.com/code4sac/trash-ai/commit/3273b243ce38d7909f7aa8768abf9b017109c3ad

File: paper.md
Comment (wincowgerDEV) [reply paper.md]: https://github.com/code4sac/trash-ai/commit/a582b32e159125913a9b0c0ab7bc996c04c9baa5

**Pull request merged**
